components/appmgt/org.wso2.carbon.appmgt.gateway/src/main/java/org/wso2/carbon/appmgt/gateway/handlers/common/SynapsePropertiesHandler.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright (c) 2005-2015, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
fixed in commit eee031731f93bdf9cb1d0f66ad611d9c0a87a972 of PR https://github.com/wso2/carbon-appmgt/pull/153